packages/paste-core/components/input-box/src/InputBox.tsx

```diff
@@ -44,19 +46,18 @@ const InputBox = React.forwardRef<HTMLDivElement, InputBoxProps>(
 
 InputBox.displayName = 'InputBox';
 
-if (process.env.NODE_ENV === 'development') {
-  InputBox.propTypes = {
-    children: PropTypes.node.isRequired,
-    disabled: PropTypes.bool,
-    hasError: PropTypes.bool,
-    insertAfter: PropTypes.node,
-    insertBefore: PropTypes.node,
-    readOnly: PropTypes.bool,
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    type: PropTypes.oneOf(['text', 'email', 'hidden', 'number', 'password', 'search', 'tel', 'date', 'time']) as any,
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    variant: PropTypes.oneOf(['default', 'inverse']) as any,
-  };
-}
+InputBox.propTypes = {
+  children: PropTypes.node.isRequired,
+  element: PropTypes.string,
+  disabled: PropTypes.bool,
+  hasError: PropTypes.bool,
+  insertAfter: PropTypes.node,
+  insertBefore: PropTypes.node,
+  readOnly: PropTypes.bool,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  type: PropTypes.oneOf(['text', 'email', 'hidden', 'number', 'password', 'search', 'tel', 'date', 'time']) as any,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  variant: PropTypes.oneOf(['default', 'inverse']) as any,
```

Comment:
So fun one on these. When ever we have a variant, we should also be passing variant down to Box, and creating a test that we can add custom styles to the variants